### ISSUE TITLE
Remove centosmsg

### ISSUE THREAD

**Issue by jpopelka** [packit/deployment#225]
tl;dr; revert #71

[centosmsg](https://github.com/packit/packit-service-centosmsg) deployment was added when we started working on CentOS Stream.
Back then, repositories were in [git.centos.org](https://git.centos.org) Pagure, which exposes events to mqtt.git.centos.org, which the centosmsg listens to.
Since then the repositories have been moved to [Gitlab](https://gitlab.com/redhat/centos-stream) and we don't serve the git.centos.org repos anymore.

Note, we still use the centosmsg in [dist2src service](https://github.com/packit/dist-git-to-source-git), but it has [its own deployment](https://github.com/packit/dist-git-to-source-git/blob/main/deployment/roles/deploy/templates/centosmsg-dc.yml.j2).
